src/content/docs/apm/agents/ruby-agent/background-jobs/sidekiq-instrumentation.mdx

```diff
@@ -27,6 +27,41 @@ By default this feature is off in case your job arguments contain sensitive info
   This feature is distinct from the generic [`capture_params`](/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration#capture_params) top-level setting, which controls whether HTTP request parameters are captured on transaction traces and traced errors for web requests. You can configure these two settings independently.
 </Callout>
 
+## Include or exclude certain job arguments [#include_or_exclude_arguments]
+
+Starting with Ruby agent version 9.5.0, an instance of the agent configured to capture Sidekiq job arguments as described above can additionally be configured to include or exclude certain arguments for fine grained control over which arguments are reported to New Relic. The `:'sidekiq.args.include'` configuration option can be set to an array of strings that will collectively serve as an allowlist. Each of those strings will be passed to `Regexp.new` for full regular expression support. For job arguments that are hashes, if a hash's key matches one of the include patterns, then both the key and its corresponding value will be included. For scalar arguments, the string representation of the scalar will need to match one of the include patterns to be captured. The `:'sidekiq.args.exclude'` option works similarly. It can be set to an array of strings that will each be passed to `Regexp.new` to create patterns. These patterns will collectively serve as a denylist for unwanted job args. Any hash key, hash value, or scalar that matches an exclude pattern will be excluded (not sent to New Relic).
```

Comment:
```suggestion
Starting with Ruby agent version 9.5.0, an agent configured to capture Sidekiq job arguments as described above can also include or exclude specific arguments for fine-grained control over which arguments are reported to New Relic. The `:'sidekiq.args.include'` configuration option can be set to an array of strings that collectively serves as an allowlist. Each of those strings will be passed to `Regexp.new` for full regular expression support. For job arguments that are hashes, if a hash's key matches one of the include patterns, then both the key and its corresponding value will be included. For scalar arguments, the string representation of the scalar will need to match one of the include patterns to be captured. The `:'sidekiq.args.exclude'` option works similarly. It can be set to an array of strings that will each be passed to `Regexp.new` to create patterns. These patterns will collectively serve as a denylist for unwanted job args. Any hash key, hash value, or scalar that matches an exclude pattern will be excluded (not sent to New Relic).
```